tidb-binlog/driver/reader/reader.go

```diff
@@ -113,22 +104,17 @@ func NewReader(cfg *Config) (r *Reader, err error) {
 		clusterID: cfg.ClusterID,
 	}
 
-	conf := sarama.NewConfig()
-	// set to 10 minutes to prevent i/o timeout when reading huge message
-	conf.Net.ReadTimeout = KafkaReadTimeout
-	if cfg.SaramaBufferSize > 0 {
-		conf.ChannelBufferSize = cfg.SaramaBufferSize
-	}
-
-	r.client, err = sarama.NewClient(r.cfg.KafkaAddr, conf)
-	if err != nil {
-		err = errors.Trace(err)
-		r = nil
-		return
-	}
+	topic, partition := cfg.GetTopic()
+	r.client, err = kafka_consumer.NewConsumer(&kafka_consumer.KafkaConfig{
+		ClientType:       cfg.ClientType,
```

Comment:
use "sarama" as default if cfg.ClientType is empty for compatibility?

Reply:
[Done](https://github.com/pingcap/tidb-tools/pull/292/files#diff-e7fa4af2efadbe7e7310025be0e2c5f1R46)